recipe/migrations/osx_arm64.txt

```diff
@@ -1060,3 +1060,4 @@ r-rcpproll
 r-desctools
 tmap
 somoclu
+cellfinder-core
```

Comment:
This needs to be the feedstock name. Not seeing any feedstock with this name currently (though maybe I'm missing something)

Reply:
Nvm GH search failed me. Now see it: https://github.com/conda-forge/cellfinder-core-feedstock